packages/google_sign_in/google_sign_in/README.md

```diff
@@ -65,6 +65,22 @@ This plugin requires iOS 9.0 or higher.
 <!-- End of the Google Sign-in Section -->
 ```
 
+As an alternative to adding `GoogleService-Info.plist` to your Xcode project, you can instead
```

Comment:
I feel like this should not be advertised as a way to replace `GoogleService-Info.plist`, it should rather be an extra feature where devs can configure the `clientID` and serverCilentId` dyanamically.
Maybe rephrase it as:
```
It is possible to configure `serverCilentId` and `clientId` dynamically in the Dart code. To do so, pass `clientId` and
`serverClientId` to the `GoogleSignIn` constructor:
```dart
GoogleSignIn _googleSignIn = GoogleSignIn(
  ...
  serverClientId: <YOUR_SERVER_CLIENT_ID>,
  ...
);
```

Reply:
Hm, I'm wondering how developers will be able to tell that `GoogleService-Info.plist` is not required when `clientId` is provided in Dart. `GoogleService-Info.plist` is a Firebase-specific file, but `google_sign_in` does not have to be restricted to use with Firebase. Flutterfire is also encouraging users to use Dart-base configuration. It seems to me that it would be confusing if the README does not make it clear that there is an option which does not require `GoogleService-Info.plist`.